components/brave_new_tab_ui/helpers/newTabUtils.ts

```diff
@@ -8,3 +8,18 @@ export const isHttpOrHttps = (url?: string) => {
   }
   return /^https?:/i.test(url)
 }
+
+/**
+ * Obtains a letter / char that represents the current site
+ * @param site - The site requested from the top site's list
+ */
+export const getLetterFromSite = (site: NewTab.Site) => {
```

Comment:
This can be a number, suggest referring to character and using `getCharForSite` or `siteToChar`

Reply:
ok updated